### DISCUSSION TITLE
Does Herd free version for Windows work with HELO

### DISCUSSION THREAD

**Discussion by Mutahhar** [beyondcode/herd-community#1173] [Q&A] [answered]
Previously, I relied on Laragon to manage my Laravel projects and used HELO to test emails. Now, I'm curious to know if the free version of Laravel Herd is compatible with HELO. If it is, could you please outline the necessary steps to set it up?

Thanks

**Comment by mpociot** [comment, marked as answer]
Yeah, that's no problem. The setup is the same :)

**Reply by Mutahhar** [reply]
Hey @mpociot! Thank you so much for your response! I appreciate it!
